File lirc-0.9.0/.install/var/run/lirc/lircd is a fifo while file lirc-0.9.0-mod/.install/var/run/lirc/lircd is a fifo
File lirc-0.9.0/.install/var/run/lirc/lircm is a fifo while file lirc-0.9.0-mod/.install/var/run/lirc/lircm is a fifo
diff -Naur lirc-0.9.0/drivers/lirc_igorplugusb/lirc_igorplugusb.c lirc-0.9.0-mod/drivers/lirc_igorplugusb/lirc_igorplugusb.c
--- lirc-0.9.0/drivers/lirc_igorplugusb/lirc_igorplugusb.c	2011-03-26 00:28:18.000000000 +0200
+++ lirc-0.9.0-mod/drivers/lirc_igorplugusb/lirc_igorplugusb.c	2012-02-26 15:27:50.477620565 +0200
@@ -88,7 +88,7 @@
 #define CODE_LENGTH	     sizeof(int)
 
 /* Igor's firmware cannot record bursts longer than 36. */
-#define DEVICE_BUFLEN	   36
+#define DEVICE_BUFLEN	   197
 
 /*
  * Header at the beginning of the device's buffer:
File lirc-0.9.0/.install/var/run/lirc/lircd is a fifo while file lirc-0.9.0-mod/.install/var/run/lirc/lircd is a fifo
File lirc-0.9.0/.install/var/run/lirc/lircm is a fifo while file lirc-0.9.0-mod/.install/var/run/lirc/lircm is a fifo
